README.md

```diff
@@ -47,7 +47,7 @@ This is a good application that grants users the ability to save list of activit
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-- [Live Demo Link](https://princebadru.github.io/To-do-list/)
+- [Not available]()
```

Comment:
- Great job making the changes requested by the previous reviewer. I noticed that this link leads to an error page. If your project has not yet been deployed, then please replace  `[Not available]()` with N/A or Coming Soon instead of including a false link in your README :
![image](https://user-images.githubusercontent.com/102699958/233781663-8f9da120-4f2a-4e32-91c4-a3db871aecd9.png)

- [ OPTIONAL ] I recommend that you remove the `Awesome Books` title at the top of your README since this is the `Leaderboard` project, and the title might cause confusion for anyone viewing your project. :+1: 

![image](https://user-images.githubusercontent.com/102699958/233781991-fc95db59-f8da-4adf-b7fd-4d254b49b17d.png)